internal/p2p/router.go

```diff
@@ -1041,3 +1058,22 @@ func (r *Router) stopCtx() context.Context {
 
 	return ctx
 }
+
+type ChannelIDs []ChannelID
```

Comment:
does this need to be exported (even internally? 

wouldn't it make sense to have this be a `map[ChannelID]struct{}` because order doesn't matter and we're really just doing membership tests and itteration.